### ISSUE TITLE
Editing a whole category of feeds? 

### ISSUE THREAD

**Issue by Aasemoon** [FreshRSS/FreshRSS#3392]
Hi there,

I was wondering, is there a way to edit a whole category of feeds at once? 
For instance, I'm subscribed to a number of feeds from the same website, and I've put them all in a category. And then something about the website is modified, and the addresses to all the feeds are slightly changed. At that point I'd have to modify all my subscriptions, and so far I only know how to do this one by one. But if there was a way for me to see / edit all the addresses in the same place, then I could just change that single parameter for all the feeds at once. So is there some way of doing this? I'd appreciate any suggestions!

P.S.: As always, thank you so much for maintaining FreshRSS, there's nothing else like it!

**Comment by Frenzie** [comment]
I don't think so right now, at least not without diving straight into SQL. Depending on what you're comfortable with it might be worth investigating programs like https://www.phpmyadmin.net/ and https://sqlitebrowser.org/

At what point referring to a program like that is a crutch for missing functionality and at what point it's highly advanced usage remains a bit of a judgment call.

If it's only a handful (let's say less than 10) it's probably no more than slightly inconvenient to edit them all individually. Additionally, I imagine a situation like that only happens very rarely.

All that being said, I wouldn't mind a (alternative?) table based management page, more akin to phpMyAdmin. You can see an example [here](https://examples.bootstrap-table.com/). But I'm not sure if it's really worth anyone's effort. :-)

**Comment by Aasemoon** [comment]
@Frenzie, Thanks for your response!
Right, I did consider editing the database directly, though as it's obviously not the best solution, I figured I'd ask here first. But if that's the only option then it'll have to do.

Oh I do have a particular use case where this happens all the time and to many many feeds [generating feeds through RSS-Bridge], so I do need to figure a way around this. =)

**Comment by aledeg** [comment]
@Aasemoon Could you explain your use-case so we could figure out if it can be beneficial to include it in the code base and how it could be useful to others? Thank you!

**Comment by Aasemoon** [comment]
@aledeg Sure. Well, I use tools such as RSS-Bridge to generate feeds for websites that don't offer them. And it happens from time to time for the parameters of the address of the generated feed to change. For instance every time a setting of the target website changes, the code that generates the feed has to be tweaked and it can result in changes in the feed address. This is the most common use-case for me, though I have also experienced situations where a news website with multiple feeds for various categories has made a change to address of all of their feeds and I've then had to re-subscribe to multiple feeds.

**Comment by aledeg** [comment]
And when you need to bulk edit them, do you mean that you need to apply the same modification on a number of feeds or to apply different modifications on a number of feeds?

**Comment by Aasemoon** [comment]
@aldeg Normally it's the same modification applied to a large number of feeds.

**Comment by descention** [comment]
@aledeg My use case would be setting category level defaults for advanced settings. I'm subscribed to 10+ individual, but distinct, feeds from a single site and keep them in the same category. In order to clean up the results for ingestion, I set article CSS selectors, CSS filters, and cookies.

I have used sqlite3 to make updates in bulk.
```
UPDATE feed
SET pathEntries = 'div[class=''content'']'
WHERE category = 2;
```
I've also used a single feed to sync attributes to the others.
```
UPDATE feed
SET attributes = (SELECT attributes FROM feed WHERE id = 11)
WHERE category = 2 AND id <> 11;
```
